apps/yapms/src/lib/components/modals/optionsmodal/OptionsModal.svelte

```diff
@@ -57,6 +58,12 @@
 					</button>
 				{/each}
 			</div>
+			<div class="form-control">
+				<label class="label cursor-pointer">
```

Comment:
We will need to decide on consistent styling for these types of checkbox inputs. Looks different in style modal. This can happen once we unify stuff from style modal into options.